src/SitemapGenerator.php

```diff
@@ -65,6 +69,19 @@ public function setMaximumCrawlCount(int $maximumCrawlCount)
         $this->maximumCrawlCount = $maximumCrawlCount;
     }
 
+    /**
+     * Enable chunk
+     *
+     * @param int $chunk
+     * @return self
+     */
+    public function setChunck(int $chunk = 50000)
```

Comment:
`chunk` sounds a bit to programmy. Could you rename this to something that expresses its function more, like `maximumNumberOfLInksPerSiteMap` or something better?